help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,142 @@
+
+![Moengage Destination Authentication](../../assets/catalog/mobile-engagement/moengage/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Region]**: Your app *data center*.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to streaming segment export destinations](../../ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from [!DNL Adobe Experience Platform] to the [!DNL Moengage] destination, you need to go through the field mapping step.
+
+Mapping consists of creating a link between your [!DNL Experience Data Model] (XDM) schema fields in your [!DNL Platform] account, and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL Moengage] destination fields, follow these steps:
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Checkbox]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/segments.png)
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/mapping.png)
+
+In the [!UICONTROL Source Field] section, select the arrow button next to the empty field.
+   
+![Moengage Destination Source Mapping](../../assets/catalog/mobile-engagement/moengage/mapping-source.png)
+
+In the [!UICONTROL Select source field] window, you can choose between two categories of XDM fields:
+* [!UICONTROL Select attributes]: use this option to map a specific field from your XDM schema to [!DNL Moengage] attribute.
+
+![Moengage Destination Mapping Source Attribute](../../assets/catalog/mobile-engagement/moengage/mapping-attributes.png)
+
+Choose your source field, then click **[!UICONTROL Select]**.
```

Comment:
again minor, but we prefer not to use the verb "click". This is for example to account for users who use alternate navigation methods than a mouse. 

Perhaps you could simply update to:

Choose your source field, then **[!UICONTROL Select]**.

Similar comment for the row below where you could replace "click the mapping icon" with "select the mapping icon"

Reply:
done